src/server/game/Maps/MapUpdater.cpp

```diff
@@ -62,12 +62,14 @@ MapUpdater::MapUpdater(): pending_requests(0)
 
 MapUpdater::~MapUpdater()
 {
-    deactivate();
+    if (activated())
+    {
+        deactivate();
+    }
 }
 
 void MapUpdater::activate(size_t num_threads)
 {
-    _workerThreads.reserve(num_threads);
```

Comment:
This was recently added here https://github.com/azerothcore/azerothcore-wotlk/commit/ff7d6d2daeb42e559841f2916d2abc9e7d429ec4 cc @lineagedr

Reply:
Should it be removed?

Reply:
@Kitzunu https://github.com/TrinityCore/TrinityCore/commit/fd47c3e06c7988de48166c5626e07dcca72bd930#diff-9c827900c952f43bc948b41600161819d2ac1a90dd80badb424f96fb7743074f :DD

Reply:
where is has-russian lable

Reply:
rofl

Reply:
someone deleted it accidentally, I added it back ;)